lib/chef/util/windows/volume.rb

```diff
@@ -18,42 +18,42 @@
 
 #simple wrapper around Volume APIs. might be possible with WMI, but possibly more complex.
 
+require 'chef/win32/api/file'
 require 'chef/util/windows'
-require 'windows/volume'
 
 class Chef::Util::Windows::Volume < Chef::Util::Windows
-
-  private
-  include Windows::Volume
-  #XXX not defined in the current windows-pr release
-  DeleteVolumeMountPoint =
-    Windows::API.new('DeleteVolumeMountPoint', 'S', 'B') unless defined? DeleteVolumeMountPoint
-
-  public
+  attr_reader :mount_point
 
   def initialize(name)
     name += "\\" unless name =~ /\\$/ #trailing slash required
```

Comment:
This is tangential to the rest of the code-review, but I just noticed that this line mutates the incoming argument.  Is this deliberate?  Then again, now I'm thinking about all the places where a constructor just holds onto a reference to an incoming string that is later mutated by the caller.  Oh Ruby...